python/GafferUI/SpreadsheetUI/_PlugTableView.py

```diff
@@ -1126,6 +1126,10 @@ def __showEditor( self, plugs, plugBound, allowDirectEditing ) :
 		self.__editorWidget = GafferUI.PlugPopup( plugs, title = "" )
 		self.__editorWidget.popup( plugBound.center() )
 
+		widget = self.__editorWidget.plugValueWidget()
+		if isinstance( widget, GafferUI.StringPlugValueWidget ) :
+			widget.textWidget().setPreferredCharacterWidth( 60 )
```

Comment:
Can you remind me when this will come into play? Just for the row names maybe? If so, perhaps it's worth using a minimum of the width of the row name field?

While we're talking about row names, we also wanted to add a "Quadruple" (and maybe a "Triple") option here : https://github.com/GafferHQ/gaffer/blob/main/python/GafferUI/SpreadsheetUI/_PlugTableView.py#L906-L910.

Reply:
> Just for the row names maybe

Yes, that is my intention. I originally included the conditional as a safety check because I wasn't sure what exactly might be getting passed to `__showEditor()`. Talking through it live, it definitely is needed because `__showEditor()` will get called for things like `_CellPlugValueWidget` as well as `StringPlugValueWidget`.

Reply:
> If so, perhaps it's worth using a minimum of the width of the row name field?

I did that and squashed it into https://github.com/GafferHQ/gaffer/pull/5484/commits/5dd723c17d254cfad3dbbc631ccbcd43d61a1f29. After we talked about it in person, I changed it to match the column width rather than the contents (with a minimum width of 250 as well) in https://github.com/GafferHQ/gaffer/pull/5484/commits/fa15f79da969592668cdcd8f5c753279880197f2.

I don't have a strong case for either being better, but having the popup match the column width is at least consistent with the cell behavior so maybe that matches user expectations better.

If agreed, I can squash that one in too.